### DISCUSSION TITLE
Local and remote clipboard images cannot be shared

### DISCUSSION THREAD

**Discussion by haoyuehongdou** [rustdesk/rustdesk#8334] [General]
Local and remote clipboard images cannot be shared. Text can be shared. Images can be copied locally through a browser, but cannot be pasted remotely. We hope this feature can be improved

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/683

**Comment by rustdesk** [comment]
ready now https://x.com/rustdesk/status/1807981430775206289
